src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -443,40 +445,25 @@ def _start_worker_agent(self) -> None:
 
         self.worker_id = self.get_worker_id()
 
-    def configure_worker_command(self, *, config: DeadlineWorkerConfiguration) -> str:
-        """Get the command to configure the Worker. This must be run as root."""
+    def configure_worker_common(self, *, config: DeadlineWorkerConfiguration) -> str:
+        """Get the command to configure the Worker. This must be run as Administrator.
+        This cannot assume that the agent user exists.
+        """
 
-        cmds = [
```

Comment:
All "install agent & setup agent user" stuff has been moved out of the base class and into the derived class.